src/routes/SurveyFeedback/component.js

```diff
@@ -0,0 +1,100 @@
+import { Component } from 'preact';
+import {Button} from '../../components/Button';
+import  { Form, Validations, FormField, TextInput } from '../../components/Form';
+import { ExperienceRating } from '../../components/Form/ExperienceRating';
+import Screen from '../../components/Screen';
+import { createClassName } from '../../components/helpers';
+import styles from './styles';
+import { ButtonGroup } from '../../components/ButtonGroup';
+
+
+export default class SurveyFeedback extends Component {
+	state = {
+		rating: { value: '' },
+		message: { value: '' },
+	}
+
+	validations = {
+		rating: [Validations.nonEmpty],
+		message: [],
+	}
+
+	getValidableFields = () => Object.keys(this.validations)
```

Comment:
The validation process has been changed, please take a look at the changes I did here in order to use the validations properly:

https://github.com/RocketChat/Rocket.Chat.Livechat/pull/419/files#diff-40c607440b11d922d97e83430f54daa8R24

Reply:
Incorporated the validation process from the suggested PR here